src/frontend/components/SøknadsSteg/EøsSteg/Søker/spørsmål.ts

```diff
@@ -0,0 +1,7 @@
+export enum EøsSøkerSpørsmålId {
+    arbeiderINorge = 'arbeider-i-norge',
```

Comment:
Litt pirk, men siden spørsmålet nå er om man arbeider eller har arbeidet, burde denne kalles bare arbeidINorge? i stedet for arbeider

Reply:
Godt poeng!